src/server/utils/astParse.js

```diff
@@ -10,6 +9,7 @@ const config = {
   sourceType: 'module',
   plugins: [
     'jsx',
+    'typescript',
```

Comment:
adding `typescript` in parser ast config

Reply:
let's duplicate into two different configs, `typescript` plugin will slowdown parsing for `javascript`